### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ser1stydent**
- [DEMO LINK](https://ser1stydent.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ser1stydent.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +12,42 @@
...
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#smartfons">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link laptops" data-qa="hover"
+             href="#laptops s computers">Laptops & Computers</a>
```
Comment (loralevitska) [inline src/index.html:35]:
```suggestion
            <a 
              class="nav__link laptops" 
              data-qa="hover"
              href="#laptops s computers"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,76 @@
...
+  --main-color:rgba(0, 172, 220, 1);
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  left: 0;
+  right: 0;
+  align-items: center;
```
Comment (loralevitska) [inline src/style.css:20]: Rewrite styles in correct orders https://9elements.com/css-rule-order/
Fix all cases

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,76 @@
...
+ }
+
+ .logo {
```
Comment (loralevitska) [inline src/style.css:23]: Fix all cases (remove redundant whitespaces)
```suggestion
.logo {
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please redeploy test report link 
image

**Comment by ser1stydent** [comment]
- [DEMO LINK](https://ser1stydent.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ser1stydent.github.io/layout_moyo-header/report/html_report/)

**Comment by ser1stydent** [comment]
- [DEMO LINK](https://ser1stydent.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ser1stydent.github.io/layout_moyo-header/report/html_report/)

File: src/index.html (hunk 1 above)
Comment (ser1stydent) [reply src/index.html:35]: - [DEMO LINK](https://ser1stydent.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ser1stydent.github.io/layout_moyo-header/report/html_report/)

**Comment by ser1stydent** [comment]
> * [DEMO LINK](https://ser1stydent.github.io/layout_moyo-header/)
> * [TEST REPORT LINK](https://ser1stydent.github.io/layout_moyo-header/report/html_repor

**Review by Esceype — CHANGES_REQUESTED** [review]
Your demo now works but the issues are still the same, fix it and don`t forget to commit them and push

**Pull request closed without merging**